domain/src/main/java/org/oppia/domain/util/StateRetriever.kt

```diff
@@ -308,10 +342,19 @@ class StateRetriever @Inject constructor(
       "FractionInput" -> InteractionObject.newBuilder()
         .setFraction(parseFraction(inputJson.getJSONObject(keyName)))
         .build()
+      "DragAndDropSortInput" -> createExactInputForDragDropAndSort(inputJson, keyName, ruleType)
       else -> throw IllegalStateException("Encountered unexpected interaction ID: $interactionId")
     }
   }
 
+  private fun parseListOfSetsOfHtmlStrings(listOfSetsOfHtmlStringsAnswer: JSONArray): ListOfSetsOfHtmlStrings {
+    val listOfSetsOfHtmlStringsBuilder = ListOfSetsOfHtmlStrings.newBuilder()
+    for (i in 0 until listOfSetsOfHtmlStringsAnswer.length()) {
+      listOfSetsOfHtmlStringsBuilder.addSetOfHtmlStrings(parseStringList(listOfSetsOfHtmlStringsAnswer.getJSONArray(i)))
```

Comment:
Please wrap this so that it fits within 100 characters. The general advice is to wrap at the 'higher syntactical element' (in this case that would be the first ``(``).

Reply:
done